core/services/pipeline/task.http.go

```diff
@@ -14,11 +14,46 @@ import (
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
+type (
```

Comment:
should we put this in the null package?

Reply:
Well, there is already a `null.Bool` but sadly it didn't work due to incompatibilities with mapstructure, that's why I opted to keep it scoped to this package.